### PR TITLE
Update Techo page to include flashing information

### PR DESCRIPTION
**Pull request by KrazyKerbalnaut**

### CODE REVIEW

**Comment by sachaw** [comment]
Please rebase and I'll merge.

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -15,6 +15,14 @@ const config = {
     colorMode: {
       respectPrefersColorScheme: true,
     },
+    announcementBar: {
+      id: 'support_us',
```
Comment (sachaw) [inline website/docusaurus.config.js:19]: Please change to something relevant "play_store_workaround" for example or omit it to use default

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -15,6 +15,14 @@ const config = {
...
+      content:
+        'Android app unavailable in the Google Play Store store. Please check out installing manually by side-loading or using the Amazon Apps method  <a href="/docs/software/android/android-installation" href="#">here</a>',
+      backgroundColor: '#ff9999',
+      textColor: '#091E42',
+      isCloseable: false,
```
Comment (sachaw) [inline website/docusaurus.config.js:24]: Should be true

File: website/docusaurus.config.js [hunk 3]
```diff
@@ -15,6 +15,14 @@ const config = {
     colorMode: {
       respectPrefersColorScheme: true,
     },
+    announcementBar: {
+      id: 'support_us',
+      content:
+        'Android app unavailable in the Google Play Store store. Please check out installing manually by side-loading or using the Amazon Apps method  <a href="/docs/software/android/android-installation" href="#">here</a>',
```
Comment (sachaw) [inline website/docusaurus.config.js:21]: I'd change the "Please check out installing manually..." Bit to "Please install from GitHub or Amazon Store. And have the text be the link.

**Review by sachaw — CHANGES_REQUESTED** [review]

**Comment by KrazyKerbalnaut** [comment]
I see that you requested changes, but I'm not able to see any of the other requests other than the 3 that you pointed out. Check those out and get back to me

File: website/docusaurus.config.js [hunk 4]
```diff
@@ -15,6 +15,14 @@ const config = {
     colorMode: {
       respectPrefersColorScheme: true,
     },
+    announcementBar: {
+      id: 'play_store_workaround',
+      content:
+        '<a href="/docs/software/android/android-installation" href="#">Android app unavailable in the Google Play Store store. Install by side-loading from git-hub or from Amazon Apps.</a>',
```
Comment (sachaw) [inline website/docusaurus.config.js:21]: I'd change it to be: 
```
Android app unavailable in the Google Play Store store. Install by side-loading from <a href="https://github.com/meshtastic/Meshtastic-Android/releases">GitHub</a> or from <a href="https://www.amazon.com/Geeksville-Industries-Meshtastic/dp/B08CY9394Q">Amazon</a>.
```

Just makes it a little more concise.

**Review by sachaw — CHANGES_REQUESTED** [review]
Last change!

**Pull request merged**